pkg/utils/getENV.go

```diff
@@ -0,0 +1,75 @@
+package utils
```

Comment:
Considering this file has the functions not only to derive OS env as passed by operator to runner but also update the engineDetails and expDetails struct accordingly with the values -- should we name this to a more generic `environment.go`?